gremlin-dotnet/example/.gitignore

```diff
@@ -0,0 +1 @@
+/Gremlin.Net.3.7.0/
```

Comment:
What is the difference between this one going into 3.6 and the one going into master? Is it to ease the difference between the package lock/go sum files? Also would this gitnore version need to match the current driver version (i.e. 3.6.x)?

Reply:
I'm not sure, both branches have been using Gremlin.Net.3.7.0 so I ignored the folder since the user can use nuget.  Should I look into changing this example to use 3.6?

Reply:
As for difference between PRs, the master PR fixes Java imports since some util packages were moved/refactored in 3.7